auto_exposure: false
color_mode: 0
exposure: 4000
readout_mode: 0
noise_filtering: false
noise_background_activity_time: 2000
